ts/features/itwallet/navigation/ItwStackNavigator.tsx

```diff
@@ -234,7 +235,14 @@ const InnerNavigator = () => {
         component={ItwPresentationCredentialFiscalCodeModal}
       />
       {/* PLAYGROUNDS */}
-      <Stack.Screen name={ITW_ROUTES.PLAYGROUNDS} component={ItwPlayground} />
+      <Stack.Screen
+        name={ITW_ROUTES.PLAYGROUNDS.MAIN}
+        component={ItwPlayground}
+      />
+      <Stack.Screen
+        name={ITW_ROUTES.PLAYGROUNDS.CREDENTIAL_DETAIL}
+        component={ItwL3CredentialDetailScreen}
+      />
```

Comment:
Do you think having a dedicated navigator for the playground might be a better approach?